examples/light-switch-app/silabs/src/LightSwitchMgr.cpp

```diff
@@ -125,6 +232,26 @@ void LightSwitchMgr::TriggerLightSwitchAction(LightSwitchAction action, bool isG
     DeviceLayer::PlatformMgr().ScheduleWork(SwitchWorkerFunction, reinterpret_cast<intptr_t>(data));
 }
 
+void LightSwitchMgr::TriggerLevelControlAction(uint8_t stepMode, bool isGroupCommand)
+{
+    BindingCommandData * data = Platform::New<BindingCommandData>();
+
+    data->clusterId = chip::app::Clusters::LevelControl::Id;
+    data->isGroup   = isGroupCommand;
+    data->commandId = LevelControl::Commands::StepWithOnOff::Id;
+    data->commandData = BindingCommandData::Step{};
+    if (auto *step = std::get_if<BindingCommandData::Step>(&data->commandData))
+    {
+        step->stepMode = static_cast<Clusters::LevelControl::StepModeEnum>(stepMode);
+        step->stepSize = LightSwitchMgr::stepSize;
+        step->transitionTime = LightSwitchMgr::transitionTime;
+        step->optionsMask = LightSwitchMgr::optionsMask;
+        step->optionsOverride = LightSwitchMgr::optionsOverride;
+    }
```

Comment:
i think this can be simplified - i'm not sure we need if check

Reply:
Yes, removed the if check and directly assigning the commandData